Description: use explicit python2 in gn
Author: Peter Michael Green <plugwash@raspbian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-08-13

--- a/tools/gn/src/gn/function_exec_script.cc
+++ b/tools/gn/src/gn/function_exec_script.cc
@@ -79,7 +79,7 @@ const char kExecScript_Help[] =
   rebase_path() function to make file names relative to this path (see "gn help
   rebase_path").
 
-  The default script interpreter is Python ("python" on POSIX, "python.exe" or
+  The default script interpreter is Python ("python2" on POSIX, "python.exe" or
   "python.bat" on Windows). This can be configured by the script_executable
   variable, see "gn help dotfile".
 
--- a/tools/gn/src/gn/setup.cc
+++ b/tools/gn/src/gn/setup.cc
@@ -762,7 +762,7 @@ bool Setup::FillPythonPath(const base::C
     }
     build_settings_.set_python_path(python_path.NormalizePathSeparatorsTo('/'));
 #else
-    build_settings_.set_python_path(base::FilePath("python"));
+    build_settings_.set_python_path(base::FilePath("python2"));
 #endif
   }
   return true;
